src/context/ContactContext/index.tsx

```diff
@@ -45,7 +47,7 @@ const ContactProvider = ({ children }: { children: ReactNode }) => {
           return false
         }
 
-        if (reg.test(contact.clientName) || reg.test(contact.pets)) {
+        if (reg.test(contact.clientName) || reg.test(contact.pets!)) {
```

Comment:
should we check if pets is not null first before passing it to another function?